community_tutorials_and_guides/blazingsql/bsql_vs_pyspark_netflow.ipynb

```diff
@@ -194,19 +194,19 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 4.11 ms, sys: 23 µs, total: 4.13 ms\n",
-      "Wall time: 3.32 ms\n"
+      "The history saving thread hit an unexpected error (OperationalError('database or disk is full')).History will not be written to the database.\n"
      ]
     },
     {
-     "data": {
-      "text/plain": [
-       "<pyblazing.apiv2.context.BlazingTable at 0x7f9230411278>"
-      ]
-     },
-     "execution_count": 5,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "NameError",
```

Comment:
Please look into the error and update the file to resolve it

Reply:
@Nicholas-7, what was the resolution on this?